tests/Unit/DataStructures/Test_DataVector.cpp

```diff
@@ -61,57 +61,74 @@ SPECTRE_TEST_CASE("Unit.DataStructures.DataVector", "[DataStructures][Unit]") {
 
 SPECTRE_TEST_CASE("Unit.DataStructures.DataVector_Ref",
                   "[DataStructures][Unit]") {
-  DataVector data{1.43, 2.83, 3.94, 7.85};
```

Comment:
Similarly to before, don't just add scopes everywhere, I have no idea if the changes you made do indeed leave the test unaltered.